setup.cfg

```diff
@@ -1,2 +1,2 @@
 [metadata]
-description-file = README.md
\ No newline at end of file
+description_file = README.md
```

Comment:
Why is this change needed ?

Reply:
![warn](https://user-images.githubusercontent.com/83482221/173761401-208eede1-78a3-460e-b634-7a1cc6dd8870.png)

Reply:
Thanks